centinel/primitives/dnslib.py

```diff
@@ -15,7 +15,7 @@ def get_ips(host, nameserver=None, record="A"):
     return lookup_domain(host, nameservers=nameservers, rtype=record)
 
 
-def lookup_domain(domain, nameservers=[], rtype="A", timeout=10):
+def lookup_domain(domain, nameservers=[], rtype="A", timeout=2):
```

Comment:
What I am asking is, do you have a response to my argument for why the timeout was initially set so high? If you agree to leaving the timeout at its original default, then we can cut your changes to the default value from this library. If you don't agree with the original timeout value, then I would ask you to respond to the previous argument.